engine/src/flutter/shell/platform/darwin/ios/framework/Source/connection_collection.h

```diff
@@ -1,36 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_IOS_FRAMEWORK_SOURCE_CONNECTION_COLLECTION_H_
-#define FLUTTER_SHELL_PLATFORM_DARWIN_IOS_FRAMEWORK_SOURCE_CONNECTION_COLLECTION_H_
-
-#include <cstdint>
-#include <map>
```

Comment:
We should have been using `std::unordered_map` which has lower overhead when ordering is not required.